clusters/clusters_api.go

```diff
@@ -531,6 +537,24 @@ func (a ClustersAPI) Create(cluster Cluster) (info ClusterInfo, err error) {
 	return
 }
 
+// Resize api can only be used when the cluster is in Running State
+func (a ClustersAPI) Resize(resizeRequest ResizeRequest) (info ClusterInfo, err error) {
+	info, err = a.Get(resizeRequest.ClusterID)
+	if err != nil {
+		return info, err
+	}
+	if info.State != ClusterStateRunning {
+		return info, fmt.Errorf("Resize API can only be called on clusters in RUNNING state. Cluster %v is in %v state", info.ClusterID, info.State)
```

Comment:
code style: error messages in Go start with lower case letter 🤷‍♂️ also try to make the lines shorter

Reply:
done